components/webgpu/lib.rs

```diff
@@ -16,18 +16,21 @@ use malloc_size_of::{MallocSizeOf, MallocSizeOfOps};
 use serde::{Deserialize, Serialize};
 use servo_config::pref;
 use smallvec::SmallVec;
+use std::ffi::CString;
 use std::ptr;
 use wgpu::{
     binding_model::{
         BindGroupDescriptor, BindGroupEntry, BindGroupLayoutDescriptor, BindGroupLayoutEntry,
     },
     id::{
         AdapterId, BindGroupId, BindGroupLayoutId, BufferId, CommandBufferId, CommandEncoderId,
-        ComputePipelineId, DeviceId, PipelineLayoutId, QueueId, ShaderModuleId,
+        ComputePipelineId, DeviceId, PipelineLayoutId, QueueId, SamplerId, ShaderModuleId,
     },
     instance::RequestAdapterOptions,
 };
-use wgt::{BufferAddress, BufferDescriptor, CommandBufferDescriptor, DeviceDescriptor};
+use wgt::{
+    BufferAddress, BufferDescriptor, CommandBufferDescriptor, DeviceDescriptor, SamplerDescriptor,
```

Comment:
Similarly, please consider using `id` as a namespace and `wgt` as a namespace.
One of the good benefits in the long run is reduced churn on these use statements: when rebasing and merging, these often come up with conflicts. Using the namespaces directly solves that.